test/models/v3/channel-parameter.spec.ts

```diff
@@ -60,10 +60,10 @@ describe('ChannelParameter model', function() {
       expect(d.hasSchema()).toEqual(true);
     });
 
-    it('should return false when there is no value', function() {
+    it('should return true when there is no value', function() {
       const doc = serializeInput<v3.ParameterObject>({});
       const d = new ChannelParameter(doc);
-      expect(d.hasSchema()).toEqual(false);
+      expect(d.hasSchema()).toEqual(true);
```

Comment:
I agree with the other change but this one is highly debatable. "hasSchema" is meant to tell you if the document has the schema specified, therefore it should be false. However, even though it's false, you should get a default schema in the "schema()" call. These are two different things IMHO.

Reply:
But for v3, you will always have a schema (underneath) as the default behavior is that the parameters are of `type: string` 🤔 Regardless of what else you define in your parameter.

For v2 it makes sense.

Reply:
As @fmvilas mentioned, hasSchema is there for answering: Did the user set the schema?

Reply:
But for v3 the schema is always set, even when an empty parameter is defined.

Reply:
What I mean is if the user has set the `schema` key. `hasSchema` answers that question. If the `schema` key is there, then `hasSchema` is true, otherwise it's false. Now, the `schema()` call will always give you a schema because it has a default. Does that make sense?

Reply:
Alright, discussed offline and this is fine. On v3, there's no `schema` keyword anymore so it should always be true.

Reply:
> But for v3, you will always have a schema (underneath) as the default behavior is that the parameters are of `type: string` 🤔 Regardless of what else you define in your parameter.
> 
> For v2 it makes sense.

Oh yeah its true😅